at_commons/lib/src/verb/update_verb_builder.dart

```diff
@@ -142,13 +146,15 @@ class UpdateVerbBuilder implements VerbBuilder {
     if (encoding != null && encoding!.isNotEmpty) {
       command += '$ENCODING:$encoding';
     }
+    if (isLocal) {
+      command += 'local:';
+    }
     if (isPublic) {
       command += 'public:';
```

Comment:
We need to make sure there are guard conditions which prevent local keys also being public. A local key **must not** be (1) public (2) shared with anyone else. Likewise, a key that is public, or shared with someone else, **must not** be local